README.md

```diff
@@ -205,6 +205,7 @@ _See also: [WeblogMatrix](https://www.weblogmatrix.org/)_
 - [Bludit](https://www.bludit.com/) `⚠` - Simple application to build a site or blog in seconds. Bludit uses flat-files (text files in JSON format) to store posts and pages. ([Demo](https://demo.bludit.com/), [Source Code](https://github.com/bludit/bludit)) `MIT` `PHP`
 - [Cadmus](https://github.com/ldemafelix/cadmus) - Cadmus is an extremely lightweight, flat-file blogging platform powered by Markdown. `MIT` `PHP`
 - [Canvas](https://trycanvas.app/) - A Laravel publishing platform. ([Source Code](https://github.com/austintoddj/canvas)) `MIT` `PHP`
+- [Castopod](https://castopod.org) - A podcast management hosting platform that includes the latest podcast 2.0 standards, an automated Fediverse feed, analytics, an embeddable player, and more. [Source Code](https://code.podlibre.org/podlibre/castopod-host) `AGPL-3.0` `PHP`
```

Comment:
Thanks for this PR! I'm the maintainer of Castopod, it would be awesome to see it added alongside other cool self-hosted projects 🙂

One thing to note: the source code url has changed since the creation of this PR.

```suggestion
- [Castopod](https://castopod.org) - A podcast management hosting platform that includes the latest podcast 2.0 standards, an automated Fediverse feed, analytics, an embeddable player, and more. [Source Code](https://code.castopod.org/adaures/castopod) `AGPL-3.0` `PHP`
```